examples/platform/silabs/efr32/rs911x/wfx_rsi_host.c

```diff
@@ -31,10 +31,10 @@
 #include "FreeRTOS.h"
 #include "event_groups.h"
 #include "task.h"
-
 #include "wfx_host_events.h"
 #include "wfx_rsi.h"
 
+
```

Comment:
No changes in this file really should probably be left as it was

Reply:
it's same file i don't know why it's showing + line